.circleci/config.yml

```diff
@@ -1,21 +1,72 @@
-version: 2.1
-
 orbs:
-  python: circleci/python@0.2.1
+  gcp-gcr: circleci/gcp-gcr@0.7.1
```

Comment:
I redid the circle configuration to use what play-store-export is using which follows the guidelines in [Building and Deploying Containers to GCR](https://docs.telemetry.mozilla.org/cookbooks/deploying-containers.html).

It's likely it's broken--I'll hone it in this PR.

Also, I commented out sections we'll want to implement in future PRs.